pinot-spi/src/main/java/org/apache/pinot/spi/config/table/TableConfig.java

```diff
@@ -101,6 +102,9 @@ public class TableConfig extends BaseJsonConfig {
   @JsonPropertyDescription(value = "Configs for Table config tuner")
   private List<TunerConfig> _tunerConfigList;
 
+  @JsonPropertyDescription(value = "Point to an existing instance partitions")
+  private Map<InstancePartitionsType, String> _instancePartitionsMap;
```

Comment:
(nit) Move this next to `_instanceAssignmentConfigMap`, same for other changes in this file